packages/ava/src/text/utils/isType.ts

```diff
@@ -0,0 +1,11 @@
+export function isString(val: unknown): val is string {
+  return typeof val === 'string';
+}
+
+export function isObject(val: unknown) {
+  return typeof val === 'object';
```

Comment:
```suggestion
  return val !== null && typeof val === 'object';
```

Reply:
what is this func for? and do we want to include nulls and arrays?

https://stackoverflow.com/a/8511350

Reply:
fixed